Mathlib/Probability/Notation.lean

```diff
@@ -24,6 +24,8 @@ We note that the notation `∂P/∂Q` applies to three different cases, namely,
 `MeasureTheory.ComplexMeasure.rnDeriv`.
 
 - `ℙ` is a notation for `volume` on a measured space.
+
+To use these notations, you need to use `open ProbabilityTheory`.
```

Comment:
```suggestion
To use these notations, you need to use `open scoped ProbabilityTheory`
or `open ProbabilityTheory`.
```

Reply:
Right, I guess it is better to tell the user to use the more restricted `open scoped` over `open`.